### ISSUE TITLE
How can i solve this issue?

### ISSUE THREAD

**Issue by SHANMUKH-CH** [filecoin-project/slate#1077]
`requests.exceptions.ConnectionError: HTTPSConnectionPool(host='uploads.slate.host', port=443): Max retries exceeded with url: /api/v3/public/ea153be1-4394-4f93-bdd4-1b2f84233f95 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000002D7179BA1F0>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))`

I'm new to slate, using the slate python api to upload a file to a collection.
When i run the script the above error is coming.

**Comment by SHANMUKH-CH** [comment]
@martinalong 
Hi, is there any way to solve this issue?

**Comment by martinalong** [comment]
Hey thanks for catching this! Looks like the endpoint wasn't linked up correctly. Try it again and let me know how that goes

**Comment by SHANMUKH-CH** [comment]
The issue persists.
`requests.exceptions.ConnectionError: HTTPSConnectionPool(host='uploads.slate.host', port=443): Max retries exceeded with url: /api/v3/public/ea153be1-4394-4f93-bdd4-1b2f84233f95 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7ff23b936670>: Failed to establish a new connection: [Errno -2] Name or service not known'))`

**Comment by martinalong** [comment]
Ok looks like the issue you were hitting was that we forgot to hook up the uploads.slate.host subdomain in a recent domain transfer we did. Just hooked that up and it should work once it propagates through (though it may take some time to reach you). The other thing I fixed earlier was something you would've encountered afterwards as well so it's good that's fixed now also. Try it again (or try in a little bit if that doesn't work) and let me know
